docs/_site/projects/docs/renderroot/light-dom.js

```diff
@@ -0,0 +1,22 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+/**
+ * This element renders its template as light DOM children.
```

Comment:
Maybe mention that <slot> cannot be used then

Reply:
Oh, `<slot>` - forgot back quotes

Reply:
Thanks!